
  std_msgs/Int64 sonar_1
  std_msgs/Int64 sonar_2
  std_msgs/Int64 sonar_3
  std_msgs/Int64 sonar_4
  std_msgs/Bool inlet
  std_msgs/Int64 encoder_1
  std_msgs/Int64 encoder_2
  std_msgs/Int64 encoder_3
  std_msgs/Int64 encoder_4
  std_msgs/Float64 rpm_1
  std_msgs/Float64 rpm_2
  std_msgs/Float64 rpm_3
  std_msgs/Float64 rpm_4
  std_msgs/Float64 pwm_1
  std_msgs/Float64 pwm_2
  std_msgs/Float64 pwm_3
  std_msgs/Float64 pwm_4
 
